src/components/discover-sheet/ListsSection.js

```diff
@@ -175,10 +178,14 @@ export default function ListSection() {
     lists,
     selectedList,
     handleSwitchList,
+    listData?.length,
   ]);
 
   const listItems = useMemo(() => {
     let items = [];
+    if (network !== networkTypes.mainnet) {
+      return items;
+    }
```

Comment:
```suggestion
    if (network !== networkTypes.mainnet) {
      return [];
    }
    let items = [];
```

nit: I think it's more clear